README.md

```diff
@@ -57,7 +59,7 @@ resources:
   source:
     uri: https://github.com/cloudfoundry-community/simple-go-web-app.git
 
-- name: resource-deploy-web-app
```

Comment:
In my experience I usually end up with a CF as one resource and multiple apps pushing to it but I understand that doesn't make sense if you're pushing across spaces. Do you think it's typical to be targeting more than one space in a pipeline? I'm not sure I understand the reasoning behind creating a resource per app.